packaging/debs/Debian/debian/control

```diff
@@ -7,11 +7,12 @@ Uploaders: Alvaro Videla <alvaro@rabbitmq.com>,
  Jean-Sébastien Pédron <jean-sebastien@rabbitmq.com>,
  Giuseppe Privitera <giuseppe@rabbitmq.com>
 Build-Depends: debhelper (>= 9),
+ dh-systemd (>= 1.5),
```

Comment:
@udf2457 I understand that. I also think we should keep in mind that not all RabbitMQ Debian package users are no Ubuntu 16.04 or will be any time soon.